Fix some bugs in Swarm<T> (#709)
planetarium/libplanet: CHANGES.md

    throws `SwarmException`.  [[#694]]
 -  `Swarm<T>` became to print less logs on debug level during sending states.
    [[#706]]

>  -  Increased `Swarm<T>`'s network timeout value, in order to stabilize

Reviewer: It would be better if we write its previous value (3s) and the current value (30s) as well.

```suggestion
 -  Increased `Swarm<T>`'s network timeout value, in order to be stable
```

>     connection in high latency environments.  [[#709]]

Reviewer: ```suggestion
    a high latency internet connection.  [[#709]]
```

 -  Fixed bug that re-download block from scratch in preloading.  [[#685]]
 -  Fixed a bug where state references become outdated if a chain is forked
    and then adding existing blocks into it.  [[#704]]

>  -  Fixed a bug where `Swarm<T>` would use all available sockets.  [[#709]]

Reviewer: ```suggestion
 -  Fixed a bug where `Swarm<T>` had exhausted all available sockets.  [[#709]]
```

>  -  Fixed a bug where `Swarm<T>` would infinitely repeat failed requests.

Reviewer: ```suggestion
 -  Fixed a bug where `Swarm<T>` had infinitely repeated failed requests.
```

    [[#709]]

>  -  Fixed a bug where `Swarm<T>` wouldn't stop properly.  [[#709]]

Reviewer: ```suggestion
 -  Fixed a bug where `Swarm<T>` hadn't stopped properly.  [[#709]]
```
